samples/scaleout/distributor-upgrade/sample.md

```diff
@@ -10,6 +10,8 @@ related:
 - samples/scaleout/distributor
 ---
 
+Note: This solution should be seen as a temporary solution. When all endpoints have been upgraded to version 6 the distributor should be migrated to [Sender Side Distribution](/transports/msmq/sender-side-distribution).
```

Comment:
Is this the new guidance? It contradicts what is written here https://docs.particular.net/transports/msmq/scaling-out